crates/sled/src/tree.rs

```diff
@@ -342,11 +368,36 @@ impl Tree {
     /// # Examples
     ///
     /// ```
-    /// use sled::{ConfigBuilder, Error};
+    /// # use sled::{ConfigBuilder, Error, IVec};
+    /// # use std::convert::TryInto;
```

Comment:
commented code still here

Reply:
No, it is a way to hide code from the generated documentation but it will be evaluated by the doc tests. It is a way to make the example more clear in some way.

Reply:
ohhhhh too much shell on my mind, I didn't know about this!

Reply:
one thing I've noticed from some of the download numbers is that a lot of people download sled and use it for personal stuff, but rarely publish crates that depend on it. I think part of this reason is because too many of the docs are way too long, and that some of them (like in the readme) are pseudocode that you can't just copy and paste. I think this is kind of an antipattern to have doc code that you can't just copy and paste and have work. could you remove the hiding code so people can copy and paste the example and it will work?

Reply:
Yeah, no problem :)

Reply:
done!